OWML.Common/IModData.cs

```diff
@@ -8,9 +8,9 @@ public interface IModData
         bool Enabled { get; }
         bool RequireVR { get; }
         bool RequireReload { get; }
+        IModConfig UserConfig { get; }
 
         void UpdateSnapshot();
         void ResetConfigToDefaults();
-        bool FixConfigs();
```

Comment:
I could keep this for backwards compatibility, but it wouldn't do anything. And I doubt it would break any mod anyway.